docs/quickstart/assessment/standalone_assessment.py

```diff
@@ -0,0 +1,111 @@
+    # which Items will be served in this assessment.
+    "activity_template_id": "quickstart_examples_activity_template_001",
+    # Uniquely identifies this specific assessment attempt session for 
+    # save/resume, data retrieval and reporting purposes. A UUID generated above.
+    "session_id": session_id,
+    # Used in data retrieval and reporting to compare results
+    # with other users submitting the same assessment.
+    "activity_id": "quickstart_examples_activity_001",
+    # Selects a rendering mode, `assess` type is a "standalone" mode (loading a
+    # complete assessment player for navigation, VS `inline`, for embedded).
+    "rendering_type": "assess",
+    # Selects the context for the student response storage. `submit_practice` 
+    # mode means student response storage in the Learnosity cloud, for grading.
+    "type": "submit_practice",
+    # Human-friendly display name to be shown in reporting.
+    "name": "Items API Quickstart",
+    # Can be set to `initial, `resume` or `review`. Optional. Default = `initial`.
+    "state": "initial"
+}
+
+# Public & private security keys required to access Learnosity APIs and
+# data. These keys grant access to Learnosity's public demos account.
+# Learnosity will provide keys for your own account.
+security = {
+    'consumer_key': config.consumer_key,
+    # Change to your domain, e.g. 127.0.0.1, learnosity.com
+    'domain': host,
+} 
+
+# Set up Learnosity initialization data.
+init = Init(
+    'items', security, config.consumer_secret,
+    request=items_request
+)
+generated_request = init.generate()
+
+# - - - - - - Section 2: your web page configuration - - - - - -#
+
+# Set up the HTML page template, for serving to the built-in Python web server
+class LearnosityServer(BaseHTTPRequestHandler):
+    def do_GET(self):
+        # Define the page HTML, as a Jinja template, with {{variables}} passed in.
+        template = Template("""<!DOCTYPE html>
+        <html>
+            <head>
+                <link rel="stylesheet" type="text/css" href="../css/style.css">
+            </head>
+            <body>
+                <h1>{{ name }}</title></h1>
+                <!-- Items API will render the assessment app into this div. -->
+                <div id="learnosity_assess"></div>
+                <!-- Load the Items API library. -->
+                <script src=\"https://items.learnosity.com/?v2021.2.LTS/\"></script>
+                <!-- Initiate Items API assessment rendering, using the signed parameters. -->
+                <script>
+                    var itemsApp = LearnosityItems.init( {{ generated_request }} );
+                </script>
+            </body>
+        </html>
+        """)
+
+        # Render the page template and grab the variables needed.
+        response = template.render(name='Standalone Assessment Example', generated_request=generated_request)
+        # Send headers and data back to the client.
```

Comment:
I'd add a new line between the rendering and the data-transmission parts.

Reply:
Fixed here: https://github.com/Learnosity/learnosity-sdk-python/commit/9c8c48d829e0c61e5a638583b8a371663810399a